scripts/mbedtls_dev/psa_storage.py

```diff
@@ -107,6 +107,14 @@ class Key:
     } #type: Dict[Expr, Expr]
     """The extendable usage flags with the corresponding extension flags."""
 
+    EXTENDABLE_USAGE_FLAGS_KEY_RESTRICTION = {
+        'PSA_KEY_USAGE_SIGN_HASH': '.*KEY_PAIR',
+        'PSA_KEY_USAGE_VERIFY_HASH': '.*KEY.*'
+    } #type: Dict[str, str]
+    """The key type filter for the extendable usage flags.
```

Comment:
I have no idea what this sentence means.

I guess the intent is to determine key types for which signature is possible? The regexps here might work now but they're fragile. It isn't the job of the `psa_storage` module to know about this since this is knowledge of cryptographic mechanisms, not of storage format. The proper place is in `crypto_knowledge.py`, and I think https://github.com/ARMmbed/mbedtls/pull/4444 adds what you need here.

I'm ok with using this as a hack for now, but please add a comment and file an issue so that we remember to fix this hack once https://github.com/ARMmbed/mbedtls/pull/4444 is merged.